lib/spear/connection.ex

```diff
@@ -113,24 +115,4 @@ defmodule Spear.Connection do
   end
 
   defp process_response(_unknown, state), do: state
-
-  defp stream_body(state, request_ref, messages) do
-    Enum.reduce_while(messages, {:ok, state}, &stream_body_message(&1, &2, request_ref))
-  end
-
-  defp stream_body_message(message, {:ok, state}, request_ref) do
-    {wire_data, _byte_size} = Request.to_wire_data(message)
-
-    stream_result =
-      Mint.HTTP.stream_request_body(
-        state.conn,
-        request_ref,
-        wire_data
-      )
-
-    case stream_result do
-      {:ok, conn} -> {:cont, {:ok, put_in(state.conn, conn)}}
-      {:error, conn, reason} -> {:halt, {:error, put_in(state.conn, conn), reason}}
-    end
-  end
```

Comment:
old buggy streaming implementation

suffered from

- not being able to emit a single event larger than an HTTP2 window
- not being able to emit lots of events together, cumulatively larger than an HTTP2 window

still nice to read though because it's so simple compared to what has replaced it